### PR TITLE
Update basics lesson to not use sbt

### PR DESCRIPTION
**Pull request by stephbian**
@matteobanerjee

### CODE REVIEW

File: web/basics.textile [hunk 1]
```diff
@@ -49,24 +49,36 @@ Scala is not just a nicer Java.  You should learn it with a fresh mind- you will
...
+scala>
+</pre>
 
-[...]
+If you don't have Maven set up, or don't have a maven project to work with, you can start the Scala REPL with <code>scala</code>.
 
-Welcome to Scala version 2.8.0.final (Java HotSpot(TM) 64-Bit Server VM, Java 1.6.0_20).
+<pre>
+$ scala
+Welcome to Scala version 2.11.4 (Java HotSpot(TM) 64-Bit Server VM, Java 1.7.0_71).
```
Comment (matteosb) [inline web/basics.textile:72]: what' up with the scala versions being different?

Comment (stephbian) [inline web/basics.textile:72]: uhhh i'm not really sure. seems that like maven is pulling a different version from somewhere.

**Comment by peckb1** [comment]
generic question: has the Scala community moved away from SBT in general, or is this mainly for our work here?

**Comment by matteosb** [comment]
Just our work here

**Comment by peckb1** [comment]
cool - thanks! :D

**Pull request merged**